### PR TITLE
learntheconcepts --> edits to smart contracts section

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

File: docs/learn_the_concepts/blockchains/smart_contracts.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
 ## What is a smart contract?
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/blockchains/smart_contracts.md:2]: infographic on smart contract workflow: "How do a smart contract work?"

File: docs/learn_the_concepts/blockchains/smart_contracts.md [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
 
-## Benefits of smart contracts
+     As a result, the transaction cannot be modified, and only parties to whom permission has been granted can view the outcome. Smart contacts, therefore, allow intermediaries such as banks and central authorities to be bypassed, as the contract is written onto the blockchain, and once written on the blockchain, these are irreversible. This means they are ideal for _automating_ and _decentralizing_ transactions and deals, and indeed are really the key to making dApps work on the blockchain.
+
+??? example
  
-There are different **positive features** of smart contracts:
+    Smart contracts are adaptable to be used in almost any sector where money, digital assets, or any type of digital information has to be moved between parties. Their application is also being investigated in the healthcare industry and to enforce intellectual property agreements.
+
+## Benefits
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/blockchains/smart_contracts.md:22]: Like the previous PRs, we clould gather benefits and drawbacks of smart contracts in a single table

File: docs/learn_the_concepts/blockchains/smart_contracts.md [hunk 3]
```diff
@@ -35,42 +37,42 @@ There are three key **negative features** about smart contracts:
 ## Use cases
 ### Finance
```
Comment (FelixNicolaeBucsa) [inline docs/learn_the_concepts/blockchains/smart_contracts.md:40]: We could add an infographic showing all possible uses of smart contracts with a brief explanation (i.e. ideally like https://research.aimultiple.com/smart-contracts-examples/)

File: docs/learn_the_concepts/blockchains/smart_contracts.md (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply docs/learn_the_concepts/blockchains/smart_contracts.md:2]: Something similar to https://vitolavecchia.altervista.org/wp-content/uploads/2022/04/Caratteristiche-e-differenza-tra-NFT-e-Smart-Contract-in-informatica.jpg could be also considered

File: docs/learn_the_concepts/blockchains/smart_contracts.md [hunk 4]
```diff
@@ -1,30 +1,32 @@
...
+!!! info
 
-## Benefits of smart contracts
+     As a result, the transaction cannot be modified, and only parties to whom permission has been granted can view the outcome. Smart contacts, therefore, allow intermediaries such as banks and central authorities to be bypassed, as the contract is written onto the blockchain, and once written on the blockchain, these are irreversible. This means they are ideal for _automating_ and _decentralizing_ transactions and deals, and indeed are really the key to making dApps work on the blockchain.
+
+??? example
  
-There are different **positive features** of smart contracts:
+    Smart contracts are adaptable to be used in almost any sector where money, digital assets, or any type of digital information has to be moved between parties. Their application is also being investigated in the healthcare industry and to enforce intellectual property agreements.
```
Comment (VijaySharma815) [inline docs/learn_the_concepts/blockchains/smart_contracts.md:19]: Their application is also being investigated in the healthcare industry and to enforce intellectual property agreements. -> Their application is also being investigated in the healthcare industry to enforce intellectual property agreements.

**Review by VijaySharma815 — APPROVED** [review]

**Pull request merged**